olp-cpp-sdk-core/include/olp/core/client/OlpClientSettingsFactory.h

```diff
@@ -58,6 +58,18 @@ class CORE_API OlpClientSettingsFactory final {
    *
    * Defaulted to platform-specific implementation.
    *
+   * @note On UNIX platforms the default network request handler is libcurl
+   * based and has the known issue of static initialization and cleanup that
+   * needs special care.  We recommend that for this you should consider
+   * initializing this network request handler at a very early stage, preferably
+   * as global static or from the main thread, and pass it on to every created
+   * client. For this matter, it also is not recommended to create multiple
+   * network request handlers.
```

Comment:
On UNIX platforms, the default network request handler is libcurl-based and has the known issue of static initialization and cleanup that needs special care.  Therefore, we recommend initializing this network request handler at a very early stage, preferably as global static or from the main thread, and pass it on to every created client. For this matter, it is also not recommended to create multiple network request handlers.